ou_dedetai/wine.py

```diff
@@ -431,62 +414,69 @@ def set_win_version(exe, windows_version):
             "/d", f"{windows_version}", "/f",
             ]
         process = run_wine_proc(
-            str(utils.get_wine_exe_path()),
+            app.conf.wine_binary,
+            app,
             exe='reg',
             exe_args=exe_args
         )
-        wait_pid(process)
-
-
-def enforce_icu_data_files(app=None):
-    repo = "FaithLife-Community/icu"
-    json_data = network.get_latest_release_data(repo)
-    icu_url = network.get_first_asset_url(json_data)
-    icu_latest_version = network.get_tag_name(json_data).lstrip('v')
+        if process is None:
+            app.exit("Failed to spawn command to set windows version for indexer")
+        system.wait_pid(process)
+
+
+# FIXME: Consider when to re-run this if it changes.
+# Perhaps we should have a "apply installation updates"
+# or similar mechanism to ensure all of our latest methods are installed
+# including but not limited to: system packages, winetricks options,
+# icu files, fonts, registry edits, etc.
+#
+# Seems like we want to have a more holistic mechanism for ensuring
+# all users use the latest and greatest.
+# Sort of like an update, but for wine and all of the bits underneath "Logos" itself
```

Comment:
These are fair comments, and certainly what the code has been building toward. But we would need some way of determining, what version of such and such winetricks command did we run, should we need to run it again.

Most of our prior validation of install was either determining a file or an existing Windows key.

The problem we get into with some of this has been, what if a user brings an install that we know nothing about? The config was the solution.